config.py

```
@@ -0,0 +1,11 @@
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv(f'.env.{os.environ.get('env')}')
```

Ruff (0.8.2)

Use capitalized environment variable `ENV` instead of `env`

(SIM112)